Formula/sgr.rb

```diff
@@ -113,13 +118,13 @@ class Sgr < Formula
   end
 
   resource "pglast" do
-    url "https://files.pythonhosted.org/packages/3b/a8/4d85067f06f13e8199dcf1efc57668e7e45160f618ae47864e98560c7b16/pglast-3.17.tar.gz"
-    sha256 "0b6496b97e8bb7847adab90d27da8ea152b9d23f93524868194c3eb1e5a34d0c"
+    url "https://files.pythonhosted.org/packages/88/df/201bb63cd9777007b89070b23e6bfb00e6da0ef2cbb9d8a4fd3df4c257e1/pglast-3.4.tar.gz"
+    sha256 "d2288d9607097a08529d9165970261c1be956934e8a8f6d9ed2a96d9b8f03fc6"
   end
```

Comment:
Hmm, why is this getting downgraded? That's almost certainly the cause of the issues.

Reply:
Yeah, doesn't make a lot of sense to me -- maybe the vulnerable version was yanked?

Reply:
I don't think pglast has a vuln?

Reply:
looks like upstream has this pinned for some reason: https://github.com/splitgraph/sgr/blob/v0.3.12/pyproject.toml#L17

We probably updated this manually. I've reverted this, and also filed https://github.com/splitgraph/sgr/pull/814 to hopefully fix this by their next release